finished-work/index.html

```diff
@@ -0,0 +1,120 @@
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="stylesheet" href="main.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,600" rel="stylesheet">
+</head>
+<body>
+  <div id="recipe1">
+  <div class="recipe-picture">
+    <h1>Apple Pies</h1>
+  </div>
+    <div id="description" class="container">
+      <p>
+        This was my grandmother's apple pie recipe. I have never seen another one
+        quite like it. It will always be my favorite and has won me several first place
+        prizes in local competitions. I hope it becomes one of your favorites as well!
+      </p>
+      <img src="/Users/raphamontenegro/Documents/ironhack/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/recipe-info.png" class="img-recipe-info">
+    </div>
+  </div>
+  <hr>
+  <div id="ingredients" class="container">
+    <h3>Ingredients</h3>
+    <ul>
+      <li>- 1 recipe pastry for a 9 inch double crust pie</li>
+      <li>- 1/2 cup unsalted butter</li>
+      <li>- 3 tablespoons all-purpose flour</li>
+      <li>- 1/4 cup water</li>
+      <li>- 3 tablespoons all-purpose flour</li>
+      <li>- 1/4 cup water</li>
+      <li>- 1/2 cup white sugar</li>
+      <li>- 1/2 cup packed brown sugar</li>
+      <li>- 8 Granny Smith apples - peeled, cored and sliced</li>
+    </ul>
+
+
+<img src="/Users/raphamontenegro/Documents/ironhack/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/cooking-info.png" class="img-recipe-info">
+</div>
+<hr>
+<section id="directions" class="container">
+  <h3>Directions</h3>
+  <ol>
+    <li>
+      Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
+      flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
+      temperature and let simmer.
+    </li>
+    <li>
+        Place the bottom crust in your pan. Fill with apples, mounded slightly. Cover with a
+        lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
+        that it does not run off.
+    </li>
+    <li>
+        Bake 15 minutes in the preheated oven. Reduce the temperature to 350 degrees F (175
+        degrees C). Continue baking for 35 to 45 minutes, until apples are soft.
+    </li> 
+  </ol>
+  </section>
+
+  <div id="recipe2">
+      <div class="recipe-picture">
+        <h1>French Onion Soup</h1>
+      </div>
+        <div id="description" class="container">
+          <p>
+              We have been trying French onion soup in restaurants for years and my family
+              and friends agree none can compare to my recipe for taste and simplicity of
+              preparation.
+          </p>
+          <img src="/Users/raphamontenegro/Documents/ironhack/lab-css-recipes-clone/starter-code/french-onion-soup-starter-code/images/recipe-info.png" class="img-recipe-info">
+        </div>
+      </div>
+      <hr>
```

Comment:
Maybe a better option instead of using ```</hr>``` would be to use something like border bottom which would take the same width of the preceding div causing you to not need to style the hr after